tests/keras/test_sequential_model.py

```diff
@@ -266,7 +266,7 @@ def test_merge_dot():
     right.add(Activation('relu'))
 
     model = Sequential()
-    model.add(Merge([left, right], mode='dot', dot_axes=1))
+    model.add(Merge([left, right], mode='dot', dot_axes=([1], [1])))
```

Comment:
I'll handle that case, if user adds single integer `i` I convert it to `([i], [i])` is that ok for you?